metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -12,6 +10,7 @@
 import traceback
 import unittest.mock
 import uuid
+from datahub.utilities._markupsafe_compat import MARKUPSAFE_PATCHED
```

Comment:
this is actually a pretty dangerous change - we're applying patches to third-party libraries, so we need to make sure that those patches get applied first

we'll need to use custom isort sections for this https://docs.astral.sh/ruff/settings/#lint_isort_sections

Reply:
also note that this is only required for `metadata-ingestion` - this specific config does not need to be copied across to every module